dotCMS/src/main/java/com/dotmarketing/quartz/job/CleanUpFieldReferencesJob.java

```diff
@@ -48,15 +49,24 @@
  * Stateful job used to remove content type field references before its deletion
  * @author nollymar
  */
-public class CleanUpFieldReferencesJob extends DotStatefulJob {
+public class CleanUpFieldReferencesJob extends DotSyncronizedTask {
 
+    final User user;
+    final Field field;
+    final Date deletionDate;
+    
+    CleanUpFieldReferencesJob(User user, Field field){
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)